threedi_modelchecker/checks/raster.py

```diff
@@ -210,6 +209,35 @@ def description(self):
         return f"The raster in {self.column_name} has non-square raster cells."
 
 
+class RasterGridSizeCheck(BaseRasterCheck):
+    """Check whether the global settings' grid size is an even multiple of a raster's cell size (at least 2x).
+
+    Only works locally.
+    """
+
+    def get_invalid(self, session):
+        grid_space_query = session.query(models.GlobalSetting.grid_space).first()
+        if grid_space_query is not None:
+            self.grid_space = grid_space_query[0]
+        else:
+            return []
+        return super().get_invalid(session)
+
+    def is_valid(self, path: str, interface_cls: Type[RasterInterface]):
+        with interface_cls(path) as raster:
+            if not raster.is_valid_geotiff:
+                return True
+            if raster.pixel_size is None:
+                return False
```

Comment:
Is this even possible? How does the `HasSquarePixels` check deal with the situation?